src/release/bin/startup.sh

```diff
@@ -5,6 +5,11 @@
 # $Id$
 # -----------------------------------------------------------------------------
 
+# Determine JAVA_HOME on Debian/Ubuntu
+if [ -z "$JAVA_HOME" ]; then
+  JAVA_HOME=$(readlink -fe /usr/bin/java | sed "s:bin/java::")
```

Comment:
This is elegant because, if the link is missing or broken, ```JAVA_HOME``` will stay empty. I like this fix because I am always annoyed by having to manually set ```JAVA_HOME``` when starting the bin.zip.

My only remark is that, in ```readlink -fe```, the "f" is redundant.